--- lab/stress_failures/survival/survival__parsecovar.R.fmt1.R	2026-02-21 14:26:07.455111661 -0600
+++ lab/stress_failures/survival/survival__parsecovar.R.fmt2.R	2026-02-21 14:26:07.455132928 -0600
@@ -347,9 +347,9 @@
         k <- seq_len(xcount[i])
         for (j in 1:ncol(tmap))
         if (tmap[i + 1, j] == 0) {
-            cmap[ii+k,j] <- 0L
+            cmap[ii + k, j] <- 0L
         } else {
-            cmap[ii+k,j] <- tmap[i + 1, j] * mult + k
+            cmap[ii + k, j] <- tmap[i + 1, j] * mult + k
         }
         ii <- ii + max(k)
     }
