--- lab/stress_failures/sp/sp__SpatialLines-methods.R.fmt1.R	2026-02-21 14:24:40.829092988 -0600
+++ lab/stress_failures/sp/sp__SpatialLines-methods.R.fmt2.R	2026-02-21 14:24:40.829111901 -0600
@@ -143,7 +143,10 @@
 setMethod("coordinates", "SpatialLines", function (obj) lapply(obj@lines, coordinates))
 
 lines.Line <- function (x, y = NULL, ...) invisible(lines(coordinates(x), ...))
-lines.Lines <- function (x, y = NULL, ...) invisible(lapply(x@Lines, function (x, ...) lines(x, ...), ...))
+lines.Lines <- function (x, y = NULL, ...) invisible(lapply(x@Lines, function (x,
+                                                                               ...) lines(
+                                                                                   x,
+                                                                                   ...), ...))
 lines.SpatialLines <- function (x, y = NULL, ...) invisible(lapply(x@lines,
         function (x, ...) lines(x, ...), ...))
 
