--- lab/stress_failures/terra/terra__plot_let.R.fmt1.R	2026-02-21 14:30:34.945907885 -0600
+++ lab/stress_failures/terra/terra__plot_let.R.fmt2.R	2026-02-21 14:30:34.945934294 -0600
@@ -894,7 +894,8 @@
 						});
 					};
 					updateLegend();
-					this.on('baselayerchange', e => updateLegend());}")
+					this.on('baselayerchange', e => updateLegend());}"
+            )
         } else if (one_legend && add_legend) {
             environment(pal)$reverse <- TRUE
             map <- leaflet::addLegend(map, position = legend, pal = pal, values = v, opacity = 1, group = nms[i],
