--- lab/stress_failures/tinytest/tinytest__tinytest.R.fmt1.R	2026-02-22 00:18:58.305701813 -0600
+++ lab/stress_failures/tinytest/tinytest__tinytest.R.fmt2.R	2026-02-22 00:18:58.305732251 -0600
@@ -714,7 +714,7 @@
         sprintf("%dh %dm %3.1fs", h, m, s)
     }
     col <- if (x < 0.1) "cyan" else "blue"
-    if (color) color_str(str, col) else str
+    if (color) { color_str(str, col) } else { str }
 }
 
 color_str <- function (x, color) {
@@ -725,7 +725,7 @@
 check_double_colon <- function (filename) {
     txt <- readLines(filename, warn = FALSE)
     i <- grepl("tinytest::expect", txt) & !grepl("#.*tinytest::expect", txt)
-    if (!any(i)) return(NULL)
+    if (!any(i)) { return(NULL) }
 
     line_numbers <- which(i)
     occurrences <- sub("^.*tinytest::expect", "tinytest::expect", txt[i])
@@ -736,7 +736,7 @@
  The results from these tests are not collected. Found the following occurrences:
 "
     issues <- sprintf("> %s#%03d: %s", basename(filename), line_numbers,
-                      occurrences)
+        occurrences)
     issues <- paste(issues, collapse = "\n ")
     postfix <- "\n Remove the 'tinytest::' prefix to register the test results."
     message(paste(prefix, issues, postfix), call. = FALSE)
@@ -749,14 +749,17 @@
     fails <- if (env$ntest() == 0) "  " # print nothing if nothing was tested
     else if (env$nfail() == 0) sprintf(if (color) "\033[0;32mOK\033[0m" else "OK")
     else sprintf(if (color) "\033[0;31m%d fails\033[0m" else "%d fails",
-                 env$nfail())
+        env$nfail())
 
     side <- if (env$nside() == 0) ""
     else sprintf(if (color) "\033[0;93m%d side-effects\033[0m " else "%d side-effects ",
-                 env$nside())
+        env$nside())
 
-    if (print) cat(prefix, fails, side, sep = " ")
-    else paste(prefix, fails, side, sep = " ")
+    if (print) {
+        cat(prefix, fails, side, sep = " ")
+    } else {
+        paste(prefix, fails, side, sep = " ")
+    }
 }
 
 #' Run all tests in a directory
@@ -848,28 +851,28 @@
         on.exit({ setwd(oldwd) ; set_call_wd("") })
         setwd(dir)
         test_output <- lapply(basename(testfiles), run_test_file,
-                              at_home = at_home, verbose = verbose,
-                              color = color,
-                              remove_side_effects = remove_side_effects, ...)
+            at_home = at_home, verbose = verbose,
+            color = color,
+            remove_side_effects = remove_side_effects, ...)
     } else {
         parallel::clusterEvalQ(cluster, library(tinytest))
         test_output <- parallel::parLapply(cluster, testfiles, run_test_file,
-                                           at_home = at_home,
-                                           verbose = min(verbose, 1),
-                                           color = color,
-                                           remove_side_effects = TRUE, ...)
+            at_home = at_home,
+            verbose = min(verbose, 1),
+            color = color,
+            remove_side_effects = TRUE, ...)
     }
 
     td <- abs(as.numeric(Sys.time()) - as.numeric(t0))
     # by using '(parL)|(l)apply' we get a list of tinytests objects. We need to unwind
     # one level to a list of 'tinytest' objects and class it 'tinytests'.
     structure(unlist(test_output, recursive = FALSE), class = "tinytests",
-              duration = td)
+        duration = td)
 }
 
 # Sort according to LC_COLLATE
 locale_sort <- function (x, lc_collate = NA, ...) {
-    if (is.na(lc_collate)) return(sort(x, ...))
+    if (is.na(lc_collate)) { return(sort(x, ...)) }
 
     # catch current locale
     old_collate <- Sys.getlocale("LC_COLLATE")
@@ -879,7 +882,8 @@
         Sys.setlocale("LC_COLLATE", lc_collate)
         TRUE
     }, warning = function (e) {
-        msg <- sprintf("Could not sort test files in 'C' locale, using %s\n", old_collate)
+        msg <- sprintf("Could not sort test files in 'C' locale, using %s\n",
+                       old_collate)
         message(paste(msg, e$message, "\n"))
         FALSE
         }, error = warning)
@@ -887,7 +891,7 @@
     out <- sort(x)
 
     # reset to old locale
-    if (colset) Sys.setlocale("LC_COLLATE", old_collate)
+    if (colset) { Sys.setlocale("LC_COLLATE", old_collate) }
     out
 }
 
@@ -992,7 +996,8 @@
 
     if (!dir.exists(testdir)) { # if not customized test dir
         # find the installed test dir
-        new_testdir <- system.file(testdir, package = pkgname, lib.loc = lib.loc)
+        new_testdir <- system.file(testdir, package = pkgname,
+                                   lib.loc = lib.loc)
         if (new_testdir == "") {
             stopf("testdir '%s' not found for package '%s'", testdir, pkgname)
         } else {
@@ -1018,14 +1023,16 @@
             parallel::clusterExport(cluster, "libs", envir = e)
             parallel::clusterEvalQ(cluster, .libPaths(libs))
         }
-        parallel::clusterCall(cluster, library, pkgname, character.only = TRUE, lib.loc = lib.loc)
+        parallel::clusterCall(cluster, library, pkgname, character.only = TRUE,
+                              lib.loc = lib.loc)
     }
 
     out <- run_test_dir(testdir, at_home = at_home, cluster = cluster, ...)
     i_fail <- sapply(out, isFALSE)
     if (any(i_fail) && !interactive()) {
         writeLines(vapply(out[i_fail], format.tinytest, "", type = "long"))
-        stop(sum(i_fail), " out of ", length(out), " tests failed", call. = FALSE)
+        stop(sum(i_fail), " out of ", length(out), " tests failed",
+             call. = FALSE)
     }
     out
 }
@@ -1102,7 +1109,9 @@
     setwd(tdir)
 
     ## build package
-    system2("R", args = c("CMD", "build", "--no-build-vignettes", "--no-manual", shQuote(pkg)))
+    system2("R",
+            args = c("CMD", "build", "--no-build-vignettes", "--no-manual",
+                     shQuote(pkg)))
 
     ## find tar.gz and install in temporary folder.
     pkgfile <- dir("./", pattern = paste0(pkgname, ".*\\.tar\\.gz"), full.names = TRUE)
