--- lab/stress_failures/data.table/data.table__setkey.R.fmt1.R	2026-02-22 00:19:56.765025635 -0600
+++ lab/stress_failures/data.table/data.table__setkey.R.fmt2.R	2026-02-22 00:19:56.765071054 -0600
@@ -418,7 +418,7 @@
                 if (length(o)) l[[i]] <- if (unique) y[o[attr(o, "starts",
                                                               exact = TRUE)]] else y[o]
                 else if (unique) l[[i]] <- y[attr(o, "starts",
-                                                  exact = TRUE)] # test 1525.5
+                        exact = TRUE)] # test 1525.5
             } else {
                 if (length(o)) { l[[i]] <- y[o] }
             }
@@ -429,7 +429,7 @@
     nrow <- prod(lengths(l))
     if (nrow > .Machine$integer.max) {
         stopf("Cross product of elements provided to CJ() would result in %.0f rows which exceeds .Machine$integer.max == %d",
-              nrow, .Machine$integer.max)
+            nrow, .Machine$integer.max)
     }
     l <- .Call(Ccj, l)
     setDT(l)
